components/Testimonial/index.vue

```diff
@@ -58,7 +60,8 @@ export default {
   async fetch () {
     const response = await this.$axios.get('/testimonials', { params: this.query })
     const { ...data } = response.data
-    this.testimonials = data
+    this.testimonials = data.data
```

Comment:
make it simplify and easy to read 

```javascript
const response = await this.$axios.get('/testimonials', { params: this.query })
const {  data, meta } = response.data
this.testimonials = data
this.meta = meta
```

Reply:
alright kak, thanks for your suggestion 🤝